Makefile

```diff
@@ -26,6 +26,23 @@ securedrop-workstation-grsec: ## Builds Debian metapackage for Qubes Workstation
 install-deps: ## Install initial Debian packaging dependencies
 	./scripts/install-deps
 
+.PHONY: requirements
+requirements: ## Creates requirements files for the Python projects
+	./scripts/create-requirements
+	./scripts/update-requirements
+
+.PHONY: build-wheels
+build-wheels: syncwheels ## Builds the wheels and syncs to the localwheels directory
+	./scripts/build-sync-wheels -r ${PKG_DIR}/requirements.txt
+	./scripts/build-sync-wheels -r ${PKG_DIR}/requirements-dev.txt
```

Comment:
we don't need this right?

Reply:
>we don't need this right?

The build-wheels action is to help us build all the dependencies into wheels. We do need this.

Reply:
sorry for lack of clarity, i meant we don't need the dev requirements files specifically, since we're not using them in the build

Reply:
>sorry for lack of clarity, i meant we don't need the dev requirements files specifically, since we're not using them in the build

Yup, and I removed that file. I forgot to update the comment here :(